include/irr/asset/filters/CMatchedSizeInOutImageFilterCommon.h

```diff
@@ -14,9 +14,27 @@ namespace irr
 namespace asset
 {
 
+/*
+	Common filter for images where input and desired output
+	image data is known. The filter can execute various
+	converting actions on input image to get an output
+	image that will be a converted and ready to use one.
+*/
+
 class CMatchedSizeInOutImageFilterCommon : public CBasicImageFilterCommon
 {
 	public:
+
+		/*
+			To make use of the filter, it's \bstate\b must be provided.
+			State contains information about input image and an image
+			that will be converted as an output, but it's reference
+			is a single choosen region data. In that case you are
+			able to convert image with various layers and faces, but
+			keep in mind that for certain mipmaps you will have to change 
+			appropriate state fields to make it work.
+		*/
```

Comment:
this documentation should be in `irr/asset/filters/IImageFilter.h` here you should be just referencing it

Reply:
okay